cpp/ql/src/Critical/MissingCheckScanf.ql

```diff
@@ -0,0 +1,119 @@
+/**
+ * @name Missing return-value check for a 'scanf'-like function
+ * @description Without checking that a call to 'scanf' actually wrote to an
```

Comment:
This is a little convoluted. Suggesting something into these lines:

`Failing to check that a call to 'scanf' actually writes to an output variable can lead to unexpected behavior at reading time`

What do you think? 🙂